docs/topics/optimizing_memory_usage.adoc

```diff
@@ -4,6 +4,9 @@
 :context: optimizing_memory_usage
 
 include::modules/reviewing_resource_information.adoc[leveloffset=+1]
-include::modules/monitor_oso_quota_usage.adoc[leveloffset=+1]
+
+//include::modules/monitor_oso_quota_usage.adoc[leveloffset=+1]
```

Comment:
I don't think we should remove this section. Instead, we should modify the two existing sections to clearly state that:

- one is for reviewing resource usage using the OSO GUI
- the other is for doing the same using the `oc` CLI

Reply:
That said, the CLI section doesn't have to be in the GSG. So, I agree with the removal.